### PR TITLE
add react-relay@0.9.0 w/ npm autoupdate

### LINKED ISSUE #8260: [Request] Add Relay
**Library name:** Relay
**Git repository url:** https://github.com/facebook/relay
**npm package url(optional):** https://www.npmjs.com/package/react-relay
**License(s):** BSD
**Official homepage:** https://facebook.github.io/relay/

### PR DESCRIPTION
**Pull request by kennynaoh**
PR for #8260 
@dannyxx001 please help me review this pull request, thank you.
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/facebook/relay
 * Official website (optional, not the repository):
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 318
   - Count of watchers: 6,515
   - Count of forks: 519
 * NPM download stats (optional): https://www.npmjs.com/package/react-relay
   - Downloads in the last day: 1,454
   - Downloads in the last week: 7,747
   - Downloads in the last month: 31,860

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8260, cc @yungsters

### CODE REVIEW

**Comment by dannyxx001** [comment]
@PeterDaveHello I think this PR is OK. Please help us review again, thanks.

File: ajax/libs/react-relay/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "react",
+    "relay"
+  ],
+  "license": "BSD-3-Clause",
+  "homepage": "https://facebook.github.io/relay/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/relay.git"
+  },
+  "author": "Timothy Yung",
```
Comment (PeterDaveHello) [inline ajax/libs/react-relay/package.json]: Should be Facebook Inc.

**Comment by kennynaoh** [comment]
@dannyxx001 
I have fixed the author field.
Please help me review it again.
Thanks!

**Comment by dannyxx001** [comment]
@PeterDaveHello I think the PR is OK. Please help us review again, thanks.

**Pull request merged**